htdocs/expedition/class/expedition.class.php

```diff
@@ -238,7 +245,7 @@ function create($user)
 		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
 		$sql.= ")";
 
-		dol_syslog(get_class($this)."::create", LOG_DEBUG);
+		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
```

Comment:
You are undoing previous work here. This is not needed anymore as it is logged by the database driver.